### DISCUSSION TITLE
NFT videos have drastically more spam comments

### DISCUSSION THREAD

**Discussion by MoralCode** [ThioJoe/YT-Spammer-Purge#509] [General]
Hello all,

In scanning some of the creators in my feed, i noticed a lot of the tech youtubers had very few spam comments (tens to low hundreds across anywhere from 5-50 videos) but when i got to a creator that recently published an NFT video, the number of spam comments shot up and I've now started scanning specifically NFT video url's and reporting WAY more spam.

So yeah, just an FYI to anyone out there thats also running this: NFT videos are a great place to find spam comments (go figure).

Does anyone agree that these kinds of tips for places to look from the community are helpful? Is it potentially worth adding a section to this discussion board to keep these "hey everyone, try looking for spam in these places" type posts now that a lot of the most common spam on recent videos is getting utterly destroyed by this script?

**Comment by KendallDoesCoding** [comment]
> 

This doesn't apply only for NFT videos, this also apply's for anything that has the following terms in the title or description:

a) robux
b) vbucks
c) anything related to bitcoin
d) money

I personally reported lots of scammers who posted domains in the above 2 sections into the YT-Spam-Lists

**Comment by MoralCode** [comment]
Wow this is an excellent list!

At this point im sorta wondering if it can be automated? i.e. search for videos with these terms (sorted by highest view count/subscriber count) using the youtube API and feed them into the script until you are out of quota, then schedule that to run daily.

**Reply by KendallDoesCoding** [reply]
this is a good idea, but it may be a little hard to code.
